public/components/marketing-forms/5.html

```diff
@@ -0,0 +1,67 @@
+<!--
+  Heads up! 👋
+
+  Plugins:
+    - @tailwindcss/forms
+-->
+
+<section class="w-full space-y-4 bg-white md:space-y-6">
+  <div>
+    <h2 class="text-2xl font-bold text-gray-900">Profile Settings</h2>
+    <p class="text-sm text-gray-700">Manage your account settings</p>
+  </div>
+  <form action="" class="border-t-2 pt-4 md:pt-6">
```

Comment:
No border color?

Reply:
I was using the default border color, but I can set `border-gray-200`